src/_guides/libraries/private-files.md

```diff
@@ -100,9 +100,11 @@ using [pub get](/guides/packages#getting-packages).
 The `pubspec.lock` file is a special case,
 similar to Ruby's `Gemfile.lock`.
 
-**For library packages**, do **not** commit the `pubspec.lock` file.
-
-**For application packages**, it's up to you whether you
-commit the `pubspec.lock` file.
-Consider checking it in if you want to track the dependencies that
-are used for production or across a team.
+**For library packages**, **don't commit** the `pubspec.lock` file.
+Recreating the `pubspec.lock` file lets you test your package
```

Comment:
`Regenerating` or simply `Generating` rather than `Recreating` seem more fitting, as `Recreating` to me seems to imply the user creates it by hand. I believe the word `generate` is used elsewhere as well.

Reply:
Good point. Thanks!